lib/elixir/src/elixir_quote.erl

```diff
@@ -395,7 +395,7 @@ do_escape(BitString, _, _) when is_bitstring(BitString) ->
       BitString;
     Size ->
       <<Bits:Size, Bytes/binary>> = BitString,
-      {'<<>>', [], [{'::', [], [Bits, {size, [], [Size]}]}, {'::', [], [Bytes, {binary, [], []}]}]}
+      {'<<>>', [], [{'::', [], [Bits, {size, [], [Size]}]}, {'::', [], [Bytes, {binary, [], nil}]}]}
```

Comment:
Without this change, `@bitstring <<"foo", 16::4>>` was generating a warning because expanded to `binary()`.
Technically, this is a breaking change, but I think it is needed for consistency?